__test__/proof/EcdsaSecp256k1RecoverySignature2020.test.js

```diff
@@ -0,0 +1,79 @@
+const {VerifiableCredential} = require('vc/VerifiableCredential');
+const {Claim} = require('index');
+const {
+    schemaLoader,
+    CVCSchemaLoader,
+} = require('index');
+const DidKeyResolver = require("../lib/util/didKeyResolver");
+const {Ed25519KeyPair} = require('@transmute/ed25519-key-pair');
+const {Secp256k1KeyPair} = require('@transmute/secp256k1-key-pair');
+
+const EcdsaSecp256k1RecoverySignature2020 = require('proof/EcdsaSecp256k1RecoverySignature2020').default;
+const {generateKeypairAndDid} = require('../lib/util/did');
+
+import {
+    ISSUER_DID,
+    SUBJECT_DID,
+    EMAIL_CREDENTIAL_IDENTIFIER,
+    EMAIL_IDENTIFIER,
+    EMAIL_DATA,
+    DOB_IDENTIFIER,
+    DOB_DATA
+} from './constants';
+
+describe('Verifiable Credentials', () => {
+    let emailClaim;
+    let dobClaim;
+    beforeAll(async () => {
+        schemaLoader.addLoader(new CVCSchemaLoader());
+
+        emailClaim = await Claim.create(EMAIL_IDENTIFIER, EMAIL_DATA);
+        dobClaim = await Claim.create(DOB_IDENTIFIER, DOB_DATA);
+    });
+
+    it('Should sign a credential using EcdsaSecp256k1RecoverySignature2020', async () => {
+        // const keypair = await Secp256k1KeyPair.generate({
+        //     secureRandom: () => Buffer.from(
+        //         "4f66b355aa7b0980ff901f2295b9c562ac3061be4df86703eb28c612faae6578",
+        //         'hex'
+        //     )
+        // });
+        //         const exportedKeyPair = await keypair.export({
+        //             type: 'EcdsaSecp256k1RecoveryMethod2020',
+        //             privateKey: true,
+        //         });
+        //
+        //         console.log(exportedKeyPair);
+
+        const keypair = {
```

Comment:
Using a hardcoded keypair - I cannot get the above generated to work (looking for a `did:key` - ot my main problem atm)